manifests/install.pp

```diff
@@ -1,26 +1,21 @@
-# == Class: sssd::install
-#
 # Install the required packages for SSSD
 #
-# == Parameters:
-#
-# [*install_user_tools*]
-# Type: Boolean
-# Default: true
-#   If true, install the 'sssd-tools' for administrative changes to the SSSD
-#   databases.
+# @param install_user_tools
+#   If ``true``, install the 'sssd-tools' package for administrative
+#   changes to the SSSD databases
 #
-# == Authors
+# @param ensure
+#   Ensure setting for all packages installed by this module
 #
-# * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
+# @author https://github.com/simp/pupmod-simp-sssd/graphs/contributors
 #
 class sssd::install (
-  Boolean  $install_user_tools = true
+  Boolean  $install_user_tools = true,
+  String   $ensure             = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'latest' }),
```

Comment:
Calling this `package_ensure` would make sure users don't think this parameter reverses the state of the rest of the resources in this class